app/ante/handler_options.go

```diff
@@ -50,6 +50,9 @@ type HandlerOptions struct {
 	DisabledAuthzMsgs []string
 	ExtraDecorators   []sdk.AnteDecorator
 	PendingTxListener PendingTxListener
+
+	// see #494, just for benchmark, don't turn on on production
+	UnsafeUnorderedTx bool
```

Comment:
do we need a flag to pass in?

Reply:
yeah, that's the app chain side of concern, normally just don't pass it at all.